spec/v0-1.md

```diff
@@ -78,17 +77,27 @@ It is an error to call `strcons` with arguments of types other than character an
 
 It is an error to call `car` or `cdr` on an argument value that is not a pair or a string.
 
+### `num`, `char`
 
-### `+`, `-`, `*`, `/`
+When `<c>` evaluates to a character, the expression `(num <c>)` evaluates to the `u32` ASCII value of the character `<c>`. 
 
-Let `+`, `-`, `*`, `/` be noted as `<op>`. The expression `(<op> <e1> <e2>)` evaluates by first evaluating `<e1>` and `<e2>` in order. The resulting values should be numbers. Then, the operation `op` is performed on these two numbers.
+When `<n>` evaluates to a Num, the expression `(char <n>)` evaluates to the character which has the ASCII value of `<n>`. If `<n>` is not a valid ASCII value, an error is thrown.
 
-Arithmetic operations are defined over a prime field, where the order is a parameter of the language.
-If the order is `P`, one can find `P - 1` by evaluating `(- 0 1)`.
+It is an error to call `num` on anything not a character and to call `char` on anything not a Num.
```

Comment:
No, it is fine to call `NUM` on a `Num` (it yields the `Num`). Also, the case of a `Comm` (commitment) is unhandled. `(num c)` returns the underlying `Num` by just changing the type tag of the `Comm` (analogous to what happens when coercing a character to number).